arch/x86_64/core/Makefile.xuk

```diff
@@ -0,0 +1,60 @@
+# Any linux host toolchain should work as a default
```

Comment:
what is this Makefile doing here?

Reply:
oh, to build basic x86_64 on Linux I guess? Can you document this somewhere?

Reply:
Exactly.  The bottom layer of the integration builds on its own with no Zephyr dependence, and was a really helpful unit test for modifying low level primitives.  Sure, will throw a README.xuk in there.